glean_parser/templates/javascript_server.jinja2

```diff
@@ -15,6 +15,9 @@ import { v4 as uuidv4 } from 'uuid';
 import mozlog{% if lang == "typescript" %}, { Logger }{% endif %} from 'mozlog';
 
 const GLEAN_EVENT_MOZLOG_TYPE = 'glean-server-event';
+{% if lang == "typescript" %}
+type LoggerOptions = { app: string; fmt?: 'heka' };
```

Comment:
```suggestion
type LoggerOptions = { app: string };
```
I'd remove format from here since ingestion pipeline expects heka and this field is overridden in line 53/56 below.

Reply:
`fmt` can be either undefined or the literal string 'heka', which is how it's used at the moment.  I guess it is not particularly helpful to the user/caller since it is always overridden as you mentioned.  But the problem is that the type checker doesn't like that `fmt` is used in the assignment without it being declared.